freqtrade/arguments.py

```diff
@@ -7,11 +7,19 @@
 import logging
 import re
 import arrow
-from typing import List, Tuple, Optional
+from typing import List, Optional, NamedTuple
 
 from freqtrade import __version__, constants
 
 
+class TimeRange(NamedTuple):
```

Comment:
Two minors comments:
1. Will it not be better to create a dedicated file for this class? It will avoid us to search in which file it is defined
2. Can we also add simple unit-test for this class?

Reply:
@glonlas Aslong as it is a NamedTuple I don't see the need to put it in a separate file, it just holds named data together, for the same reason I think its not necessary to write tests for it. All of the logic happens in `parse_timerange` which is already tested.